easysocial/easysocial/group.php

```diff
@@ -83,6 +81,13 @@ public function post()
 		$apiResponse->result	=	new stdClass;
 		$postValues				=	$input->post->getArray();
 
+		// Check EasySocial extension version
+		$component = JComponentHelper::getComponent('com_easysocial');
+		$extension = JTable::getInstance('extension');
+		$extension->load($component->id);
```

Comment:
@Pratikshakhandagale Easysocial has their own method to check the version. can you use that